dist/listbox/ds4/listbox.css

```diff
@@ -46,6 +46,9 @@ div.listbox__option[role="option"]:hover {
 div.listbox__option[role="option"]:active {
   font-weight: normal;
 }
+div.listbox__option[role="option"]:active .menu-button__status,
```

Comment:
I think something has gone wrong here. We shouldn't be seeing menu styles in the listbox.css.

Reply:
Yes I mentioned that in description. 
The mixin for dropdown has nested .listbox__status styles which will be pulled both in listbox and menu. I added menu-button status styles too in order to follow that, but it creates these extra styles. 
What should probably be done is create a generic status style to apply the styles from mixin to. Otherwise to move those styles in their respective classes

Reply:
Okay, we should try and fix this. For now let's go the simplest, less disruptive route which is probably to just move the styles into their respective classes (as you suggest).